pyqtgraph/colormap.py

```diff
@@ -115,7 +135,7 @@ def _getFromFile(name):
     return cm
 
 def getFromMatplotlib(name):
-    """ import colormap from matplotlib definition """
+    """ Generates a ColorMap object from a Matplotlib definition. Same as ``colormap.get(name, source='matplotlib')``. """
```

Comment:
should this perhaps be 

```
:method:~`pyqtgraph.colormap.get`
```

that way we get links within the docs?

Reply:
actually, scratch that, you're demonstrating an equivalent function, not looking to reference one...